modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/context/UADExportProcessDisplayContext.java

```diff
@@ -215,16 +215,12 @@ portletRequest, getPortletURL(), null,
 							themeDisplay.getScopeGroupId(),
 							selectedUser.getUserId(), status);
 
-					results = ListUtil.sort(
+					return ListUtil.sort(
 						results,
 						getComparator(
 							themeDisplay.getLocale(),
 							searchContainer.getOrderByCol(),
 							searchContainer.getOrderByType()));
-
-					return ListUtil.subList(
```

Comment:
@JanisZhang It seems the simplification in https://github.com/liferay-core-infra/liferay-portal/pull/4255/commits/6364ec9574a88de2dd6285c6e161cebdd7bce62a is correct, but this commit is wrong.

```
	public void setResultsAndTotal(List<R> results) {
		_setTotal(results.size());

		_setResults(results.subList(_start, _resultEnd));
	}

	public <E extends Throwable> void setResultsAndTotal(
			UnsafeSupplier<List<R>, E> setResultsSupplier, int total)
		throws E {

		_setTotal(total);

		_setResults(setResultsSupplier.get());
	}
```

The lambda version of the method doesn't `subList` the input.